### ISSUE TITLE
Zeek renaming: Update Plugin namespace

### ISSUE THREAD

**Issue by rsmmr** [zeek/zeek#323]
It'd be good to update anything user-visible relating to plugins so that we avoid breaking stuff there later.

I can think of the following right now:

- The built-in plugins are all in the Bro namespace:

```
# bro -N
Bro::ARP - ARP Parsing (built-in)
Bro::AsciiReader - ASCII input reader (built-in)
...
```

- CMake support is in `BroPlugin`

- The plugin skeleton in aux has various Bro references

**Comment by jsiwek** [comment]
Doing this is no longer planned for 3.0.  The thinking is that there will be a more thorough redefinition of the plugin API later on (e.g. setting some limitations around things plugins can use/access), so any Bro-to-Zeek renaming could potentially happen at the that same time.

**Comment by jsiwek** [comment]
Last comment about waiting to change plugin API internals (C++ and CMake infrastructure) still stand, but think this ticket can be closed via merging #404, which IMO takes care of the last blatantly user-visible aspect of the Bro renaming with respect to plugins.
